src/linux/classes/com/alibaba/wisp/engine/WispCarrier.java

```diff
@@ -145,6 +145,7 @@ final WispTask runTaskInternal(Runnable target, String name, Thread thread, Clas
         } finally {
             isInCritical = isInCritical0;
         }
+        wispTask.enterTs = System.nanoTime();
```

Comment:
enable the feature by default?

Reply:
yes. We could add an environment variable for that since it is not so risky.
Also, use a profile flag to control all MxBeans(Wisp and Thread) is also a good idea. I think..